### PR TITLE
Docs

### LINKED ISSUE #136: Update README
We should update the README now. The changes could involve a better title, some information about CoC, and details about `client` and `new_client`.

### PR DESCRIPTION
**Pull request by ShubhankarKG**
Documentation for the project.

1. Cache.md
2. Setup.md
3. Contributing.md
4. LICENSE

Fixes #136

### CODE REVIEW

**Comment by ShubhankarKG** [comment]
Any thoughts on content for README? @Saif807380 @akshatshah21

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+In short, when you submit code changes, your submissions are understood to be under the same [GNU GPL 3.0 License](https://choosealicense.com/licenses/gpl-3.0/) that covers the project. Feel free to contact the maintainers if that's a concern.
+
+## Report bugs using Github's [issues](https://github.com/CommunityOfCoders/COCWebsite/issues)
+We use GitHub issues to track public bugs. Report a bug by [opening a new issue](https://github.com/CommunityOfCoders/COCWebsite/issues/new); it's that easy!
+
+## Use a Consistent Coding Style
+
+We have prettier in the client-side, and we prefer to have a consistent code format.
+
+* You can try running `npm run format` or have `husky` done that for you in a pre-commit git hook for style unification.
```
Comment (Saif807380) [inline CONTRIBUTING.md:30]: have `husky` do that for you

File: SETUP.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+# SETUP
+
+To setup a local development environment, follow the steps given below.
+
+1. Ensure you have NodeJS, npm and redis installed in your system.
+2. Clone the repository
```
Comment (Saif807380) [inline SETUP.md:6]: Fork and Clone maybe?

File: SETUP.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+```bash
+$ git clone https://github.com/CommunityOfCoders/COCWebsite.git
+```
+3. Install dependencies in `new_client` folder.
```
Comment (Saif807380) [inline SETUP.md:10]: Can we change the name from `new_client` to `client`?

**Review by Saif807380 — CHANGES_REQUESTED** [review]
Good Work!

**Comment by ShubhankarKG** [comment]
Merging this one!

**Pull request merged**